packages/webapp-spa-server/package.json

```diff
@@ -31,7 +31,7 @@
     "compression": "^1.7.4",
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
-    "express": "^4.18.2",
+    "express": "^4.19.0",
```

Comment:
Shouldn't the version to remediate be 4.19.2?

https://github.com/advisories/GHSA-rv95-896h-c2vc

Reply:
I didn't pay attention to the ^.